app/liquid_tags/null_tag.rb

```diff
@@ -4,7 +4,7 @@ def initialize(tag_name, _markup, _options)
   end
 end
 
-disabled_tags = %w[assign break capture case comment cycle decrement for if ifchanged include increment unless tablerow]
+disabled_tags = %w[assign break capture case cycle decrement for if ifchanged include increment unless tablerow]
```

Comment:
Just a side note, this is here to disable Liquid's build-in [comment feature](https://shopify.github.io/liquid/tags/comment/) for security reasons. Technically it would now be overwritten by our "dev comment" so It shouldn't be a problem.